### PR TITLE
Updates for CMO 2018

### PR DESCRIPTION
**Pull request by speedy1812**
This is a PR to update the site with the necessary info for announcing CMO 2018.

### CODE REVIEW

File: data/requirements.yml [hunk 1]
```diff
@@ -5,14 +5,14 @@
...
+- title: "Commitment of Service"
+  body: "<p>CMO %{current_project_year} will begin on %{project_start_date} and last through %{project_end_date}. We ask that all young men participating in CMO to come for the duration of the project. It is not practical to come for less.</p><p>All team members are expected to arrive on %{max_arrival_date}. If you think you will have difficulty arranging your travel plans to concide with this date, please <a href=\"mailto:info@euroteamoutreach.org\">contact us</a> to discuss other options. More detailed travel information is available in the <a href=\"/downloads\">Info Pack</a>.</p><p>If you are a married man or a church leader, and feel that you would not be able to stay for the whole project, please <a href=\"mailto:info@euroteamoutreach.org\">contact us</a> about this. We recognize that for some, a shorter stay is the only option.</p>"
```
Comment (joshukraine) [inline data/requirements.yml]: I think we have a problem with this sentence: _We ask that all young men participating in CMO to come for the duration of the project._

Need to remove word "to": _We ask that all young men participating in CMO ~to~ come for the duration of the project._

File: helpers/custom_helpers.rb [hunk 2]
```diff
@@ -1,22 +1,30 @@
...
+    "July 12"
   end
 
   def operations_fee
-    "140"
+    "650"
+  end
+
+  def personal_support
+    "550"
```
Comment (joshukraine) [inline helpers/custom_helpers.rb]: In your annotated PDF you have $600. Are we going with $550 then?

File: source/apply.html.haml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Apply for CMO 2016
+title: Apply for CMO
```
Comment (joshukraine) [inline source/apply.html.haml]: Since we still have the current project year displayed in the hero unit, let's interpolate here the same way.

Comment (speedy1812) [reply source/apply.html.haml]: Interpolation didn't work here, so we'll just leave it as is for now.

**Comment by speedy1812** [comment]
@joshukraine I checked, and all the page variables are good to go. Just need to upload the docs, and I think we will be ready to merge.

**Pull request merged**